godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/sensors_rgbd.yaml

```diff
@@ -0,0 +1,31 @@
+sensors:
+  - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
+    # TODO: use appropriate cloud input topic
+    point_cloud_topic: godel_octomap_input
+    max_range: 5.0
+    point_subsample: 1
+    padding_offset: 0.05
+    padding_scale: 1
```

Comment:
The padding parameters are also just place-holders: proper values will have to be determined.

Whether self-filtering should be enabled is also unsure.